lib/Migration/Version000000Date20220316165602.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\BigBlueButton\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+/**
+ * Auto-generated migration step: Please modify to your needs!
+ */
+class Version000000Date20220316165602 extends SimpleMigrationStep {
```

Comment:
Since both db modifications belong to one feature, it would be nice if you could merge them to one migration.